scrapy/commands/parse.py

```diff
@@ -7,7 +10,9 @@
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
 from scrapy.exceptions import UsageError
-from scrapy import log
+
+logger = logging.getLogger('scrapy')
```

Comment:
We can also use a custom Formatter which uses [scrapy] for all our loggers, stripping the hierarchy, but keeps other names untouched. Or a default format string which just uses [scrapy] regardless of logger name - this formatter can be attached only to scrapy logger, not to user logs.